openwisp_monitoring/check/tasks.py

```diff
@@ -35,22 +35,43 @@ def perform_check(uuid):
 
 
 @shared_task
-def auto_create_ping(model, app_label, object_id, created):
+def auto_create_ping(model, app_label, object_id):
```

Comment:
I think it's safe to remove the the keyword argument `created` , since it's not being used at all.